src/components/error-summary/template.test.js

```diff
@@ -0,0 +1,186 @@
+
+describe('Error-summary', () => {
+  it('aria-labelledby attribute matches the title id', () => {
+    const { $ } = render('error-summary', examples.default)
+    const ariaAttr = $('.govuk-c-error-summary').attr('aria-labelledby')
+
+    expect(ariaAttr).toEqual('error-summary-title')
+  })
+
+  it('has role=alert attribute', () => {
+    const { $ } = render('error-summary', examples.default)
+    const roleAttr = $('.govuk-c-error-summary').attr('role')
+
+    expect(roleAttr).toEqual('alert')
+  })
+
+  it('has the correct tabindex attribute to be focussed', () => {
+    const { $ } = render('error-summary', examples.default)
+    const tabindexAttr = $('.govuk-c-error-summary').attr('tabindex')
+
+    expect(tabindexAttr).toEqual('-1')
+  })
+
+  it('renders title text', () => {
+    const { $ } = render('error-summary', examples.default)
+    const summaryTitle = $('.govuk-c-error-summary__title').text().trim()
+
+    expect(summaryTitle).toEqual('Message to alert the user to a problem goes here')
+  })
+
+  it('allows title text to be passed whilst escaping HTML entities', () => {
+    const { $ } = render('error-summary', {
+      titleText: 'Alert, <em>alert</em>'
+    })
+
+    const summaryTitle = $('.govuk-c-error-summary__title').html().trim()
+    expect(summaryTitle).toEqual('Alert, &lt;em&gt;alert&lt;/em&gt;')
+  })
+
+  it('allows title HTML to be passed un-escaped', () => {
+    const { $ } = render('error-summary', {
+      titleHtml: 'Alert, <em>alert</em>'
+    })
+
+    const summaryTitle = $('.govuk-c-error-summary__title').html().trim()
+    expect(summaryTitle).toEqual('Alert, <em>alert</em>')
+  })
+
+  it('renders description text', () => {
+    const { $ } = render('error-summary', examples.default)
+    const summaryDescription = $('.govuk-c-error-summary__body p').text().trim()
+
+    expect(summaryDescription).toEqual('Optional description of the errors and how to correct them')
+  })
+
+  it('allows description text to be passed whilst escaping HTML entities', () => {
+    const { $ } = render('error-summary', {
+      descriptionText: 'See errors below (▼)'
+    })
+
+    const summaryDescription = $('.govuk-c-error-summary__body p').html().trim()
+    expect(summaryDescription).toEqual('See errors below (&#x25BC;)')
+  })
+
+  it('allows description HTML to be passed un-escaped', () => {
+    const { $ } = render('error-summary', {
+      descriptionHtml: 'See <span>errors</span> below'
+    })
+
+    const summaryDescription = $('.govuk-c-error-summary__body p').html().trim()
+    expect(summaryDescription).toEqual('See <span>errors</span> below')
+  })
+
+  it('allows additional classes to be added to the error-summary component', () => {
+    const { $ } = render('error-summary', {
+      classes: 'extra-class one-more-class'
+    })
+
+    const $component = $('.govuk-c-error-summary')
+    expect($component.hasClass('extra-class one-more-class')).toBeTruthy()
```

Comment:
I am surprised this works - it seems to go against Cheerio and jQuery's documentation which both expect a single class to be passed. Wouldn't be surprised if this is a bug in Cheerio…